recipe/meta.yaml

```diff
@@ -25,10 +25,11 @@ requirements:
     - param >=1.6.1,<2.0
     - numpy >=1.0
     - matplotlib >=2.1
-    - bokeh >=0.12.15
+    - bokeh >=0.12.15,<=0.12.16
     - jupyter
     - notebook
     - ipython >=5.4.0
```

Comment:
We may also consider pushing this constraint upstream as well (not that it changed in this PR).

ref: https://github.com/ioam/holoviews/blob/v1.10.5/setup.py#L17

Reply:
That constraint is expressed in the [``meta.yaml``](https://github.com/ioam/holoviews/blob/master/conda.recipe/meta.yaml) which I used to update this recipe with ``conda render``. I agree we should update ``setup.py`` too so I'll make a holoviews PR for that.

Reply:
Do we really need to update setup.py to include this constraint now if we want to remove it as soon as next week?

Reply:
I was thinking of adding the updated constraint next week so it is ready in time for the next conda-forge recipe update.

Reply:
Okay makes sense, that would be ``bokeh >=0.12.15,<=0.13.0``?

Reply:
I assume so, yes.